rfcs/rfc-manage-staff-role-and-account.md

```diff
@@ -0,0 +1,75 @@
+- Start Date: (2022-07-23)
+- Target Major Version: N/A
+- Reference Issues: (fill in existing related issues, if any)
+- Entity Issue: https://github.com/bcgov/entity/issues/12226
+- Implementation PR: None
+
+
+# Summary
+
+SBC staff need to access the staff dashboard with limited privileges to support SP/GP filing/alteration.
+
+This is an RFC draft to discuss more suitable approaches 
+
+# Basic example
+
+If the proposal involves a new or changed API, component, etc., include a basic code example.
+Omit this section if it's not applicable.
+
+# Motivation
+SBC staff want to access the staff dashboard to search for SP/GP filing for clients, They may have to register SP/GP companies on behalf of the client
+
+# Solution Options
+## 1) Use a new role for sbc staff from keycloak
+
+### Summary
+Keycloak Roles are currently used across the application to determine who can access the staff dashboard.
+We can create a new keycloak role for sbc-staff and in code, we can use this group to give permission for required components/pages.
+In this way, sbc-staff can do alterations for the organization on behalf of the user, and sbc-staff can use their GOVM account to create new SP/GP for Users
+We can create a new GOVM account and add all BC Registry staff into that GOVM account where our staff can access the products as sbc-staffs do.
+
+### Detailed design
```

Comment:
Could this show a flow, or design for the implementation, as it mixes in adoption issues (eg. code we have to check at all places where we need to give permission)